content/filters/string_filters.md

```diff
@@ -0,0 +1,273 @@
+# String Filters
+
+String filters are used to manipulate outputs and variables of the [string](/) type.
```

Comment:
outputs and variables of the string type?
I would remove the link, not sure what this type section would be now